src/Nixfmt/Pretty.hs

```diff
@@ -81,7 +82,7 @@ instance Pretty Trivium where
     comment (if isDoc then "/**" else "/*")
       <> hardline
       -- Indent the comment using offset instead of nest
-      <> offset 2 (hcat $ map prettyCommentLine c)
+      <> offset indentWidth (hcat $ map prettyCommentLine c)
```

Comment:
I'm a bit unsure about the `offset` here. I know that it used to allow odd numbers in the past (due to the way comments were styled) prior to some refactorings, but currently it may the case that all offsets are just indentation levels. In that case the `offset` function should be redefined to only track the indentation *level*, and then multiply that with the indentation width only during rendering.